schemes.md

```diff
@@ -57,16 +62,32 @@ technology used by a given screen.
 
 ## Extensibility
 
-The mechanism of using schemes in the OSP allows to extend the supported
-application types on receiver devices. For example Android TV can implement the
-OSP (receiver part) for native TV applications in the future e.g. using the
-scheme `android` (the TV App uses in this case a native library that implements
-the OSP with similar interface to Presentation API). There is no need to
-update/extend the controlling UA to support new schemes, but it is up to the
-vendor of controlling UA to white/blacklist specific URL schemes.
-
-*TODO:*
-[Issue #93: Custom schemes and interop](https://github.com/webscreens/openscreenprotocol/issues/93)
+The mechanism of using schemes in the OSP allows controlling pages to support
+additional receiver application types.  For example, Android TV could implement
+the OSP receiver for native TV applications in the future and advertise them
+using the scheme `android.` A native Android TV app with a native library that
+acts as a presentation receiver could then be controlled through the
+Presentation Controller API.
```

Comment:
I'd like to add another example: "A controlling page could open an HbbTV web application on the OSP receiver and, but using an `hbbtv` URI scheme pass additional parameters that enable the receiving web page to access broadcast-specific functionality"

Reply:
Done, I lightly edited your suggestion, please let me know if that fits what you expected.